shared/style/value_selector.css

```diff
@@ -4,7 +4,7 @@
 
 /* Main dialog setup */
 [role="dialog"][data-type="value-selector"] {
-  background:rgb(243, 243, 243);/* UX:6.4.1 #2d2d2d --> white*/
+  background:#565050;/* UX:6.4.1 #2d2d2d --> white*/
```

Comment:
fix the comment. #565050 is not a white color as far as I know.
Also, you can set it as rgb(86, 80, 80) instead of #565050